src/scripts/components/component.es6

```diff
@@ -96,7 +96,7 @@ module.exports = class Component {
 	destroy() {
 		this.detachListeners()
 
-		for (let prop in this) {
+		for (const prop in this) {
```

Comment:
Za mě `const`. Situace, že by si někdo rozbil `for (...; ...; ...)` kvůli `const` je snadno odhalitelná.